<launch>
  <node pkg="move_arm" type="move_arm" output="screen">    

    <remap from="robot_description" to="robotdesc/pr2" />
    <remap from="collision_map" to="collision_map_occ" />
    <remap from="collision_map_update" to="collision_map_occ_update" />

    <remap from="arm_ik" to="pr2_ik_node/ik_service" />	
    <remap from="arm_ik_query" to="pr2_ik_node/ik_query" />
    
    <remap from="controller_start"  to="l_arm_joint_trajectory_controller/TrajectoryStart" />
    <remap from="controller_query"  to="l_arm_joint_trajectory_controller/TrajectoryQuery" />
    <remap from="controller_cancel" to="l_arm_joint_trajectory_controller/TrajectoryCancel" />
    
    <remap from="motion_plan" to="plan_kinematic_path" />
    
    <param name="arm" type="string" value="left_arm" />
    <param name="refresh_interval_collision_map" type="double" value="5.0" />
    <param name="refresh_interval_kinematic_state" type="double" value="1.0" />
    <param name="bounding_planes" type="string" value="0 0 1 -0.01" />
  </node>
</launch>
